pyaedt/modeler/GeometryOperators.py

```diff
@@ -1448,3 +1462,354 @@ def orient_polygon(x, y, clockwise=True):
+            If ``False`` the left-handed rotation from Va to Vb is considered. The default is ``True`.
+
+        Returns
+        -------
+        float
+            Angle in radians.
+
+        """
+        tol = 1e-12
+        cross = GeometryOperators.v_cross(va, vb)
+        if GeometryOperators.v_norm(cross) < tol:
+            return math.pi
+        assert GeometryOperators.is_collinear(cross, vn), (
+            "vn must be the normal to the " "plane containing va and vb."
+        )  # pragma: no cover
+
+        vnn = GeometryOperators.normalize_vector(vn)
+        if righthanded:
+            return math.atan2(GeometryOperators.v_dot(cross, vnn), GeometryOperators.v_dot(va, vb))
+        else:
+            mcross = GeometryOperators.v_cross(vb, va)
+            return math.atan2(GeometryOperators.v_dot(mcross, vnn), GeometryOperators.v_dot(va, vb))
+
+    @staticmethod
+    @aedt_exception_handler
+    def v_angle_sign_2D(va, vb, righthanded=True):
+        """Evaluate the signed angle between two 2D geometry vectors.
+        Iit the 2D version of the ``GeometryOperators.v_angle_sign`` considering vn = [0,0,1].
+        In case of opposite vectors, it returns an angle equal to 180deg (always positive).
+
+        Parameters
+        ----------
+        va : list
+            List of ``[x, y]`` coordinates for the first vector.
+        vb : list
+            List of ``[x, y]`` coordinates for the second vector.
+
+        righthanded : bool
+            If ``True`` the right-handed rotation from Va to Vb is considered.
+            If ``False`` the left-handed rotation from Va to Vb is considered. The default is ``True`.
+
+        Returns
+        -------
+        float
+            Angle in radians.
+
+        """
+        c = va[0] * vb[1] - va[1] * vb[0]
+
+        if righthanded:
+            return math.atan2(c, GeometryOperators.v_dot(va, vb))
+        else:
+            return math.atan2(-c, GeometryOperators.v_dot(va, vb))
+
+    @staticmethod
+    @aedt_exception_handler
+    def point_in_polygon(point, polygon):
+        """
+        Determine if a point is inside or outside a polyogn, both located on the same plane.
+        The method implements the radial algorithm (https://es.wikipedia.org/wiki/Algoritmo_radial)
+
+        point : list
+            List of ``[x, y]`` coordinates.
+        polygon : list
+            [[x1, x2, ..., xn],[y1, y2, ..., yn]]
+
+        Returns
+        -------
+        int
+            -1 When the point is outside the polygon.
+            0 When the point is exactly on one of the sides of the polygon.
+            1 When the point is inside the polygon.
+        """
+        # fmt: off
+        tol = 1e-8
+        assert len(point) == 2, "point must be a list in the form [x, y]"  # pragma: no cover
+        pl = len(polygon[0])
+        assert len(polygon[1]) == pl, "Polygon x and y lists must be the same length"  # pragma: no cover
```

Comment:
Avoid using `assert`. Instead use `raises` and then include this in unit testing.

Reply:
I've changed using ``raise``, but it's not possible to unit test. Testing with pytest is possible, but then it will fail the Ironpython test.